prow/plugins/approve/approve.go

```diff
@@ -171,6 +172,63 @@ func handleGenericComment(log *logrus.Entry, ghc githubClient, repo approvers.Re
 	)
 }
 
+// handleReviewEvent should only handle reviews that have no approval command.
+// Reviews with approval commands will be handled by handleGenericCommentEvent.
+func handleReviewEvent(pc plugins.PluginClient, re github.ReviewEvent) error {
```

Comment:
Did we notify the mailing lists?

Reply:
I haven't notified any mailing lists. What should the notification be?